### ISSUE TITLE
Doctest Regression in 4.5.  All tests that previously executed and passed, are now "Skipped"

### ISSUE THREAD

**Issue by richyo-tessonics** [matepek/vscode-catch2-test-adapter#392]
**Checklist**

- [x] The issue is about this extension and NOT about a fork.
- [x] Check the [known issues](https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/support.md#known-issues) list.
- [x] The latest version of the extension was used.
- [x] It is not related to remote-vscode or I checked the following [issue](https://github.com/matepek/vscode-catch2-test-adapter/issues/201)
- [x] Imagine yourself into my position and think how hard to debug the issue without insufficient information.
      I understand that you have privacy concerns and I expect you to understand that this extension is developed for free.
      Thanks.

**Describe the bug**

Since version 4.5, all my doctests continue to be discovered, however attempting to run them now longer shows pass or fail.  They all show skipped

**To Reproduce**
```
#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN 
#include "doctest.h"

TEST_CASE("first test") {

}


TEST_CASE("second test") {

}
```

![image](https://github.com/matepek/vscode-catch2-test-adapter/assets/126701490/b1bd1e23-3812-45c9-b4a7-f4589b32149f)

**Desktop**

- Extension Version:
- VS Code Version: 1.80
- Catch2 / Google Test / DOCTest Version:
- OS Type and Version: Windows 10, Windows 11
- Using remote-ssh/docker/wsl?: no

**Regression bug?**
The previous release 4.4 works.  The doctest bug fixes in 4.5 are causing a regression for me.

I've validated this on Windows 10, and Windows 11

**Log** (optional but recommended)

Note I will follow up with more information

**Comment by lostfalter** [comment]
I have the same issue on the Mac.

**Comment by matepek** [comment]
<details>
<summary>Fixed in <b>v4.5.1</b>.</summary>

This issue was mentioned in [CHANGELOG.md](./CHANGELOG.md) under a released entry so it is assumed to be fixed.
User verifications are always welcome.
</details>
